frontend/src/components/buttons/AddRowButton.tsx

```diff
@@ -29,13 +31,16 @@ export default function AddRowButton<T extends IDer>(
   };
 
   return (
-    <Button
-      type="button"
-      label="New"
-      icon="pi pi-plus"
-      className="p-button-info mr-2"
-      onClick={addNewRow}
-      disabled={!!props.isModifiable}
-    />
+    <>
+      <Button
+        type="button"
+        label={props.label ?? "Add"}
+        icon="pi pi-plus"
+        className="p-button-info mr-2"
+        onClick={addNewRow}
+        disabled={props.isDisabled}
```

Comment:
do we need ?? defaults or no

Reply:
We actually don't, but I'm going to add them for clarity

Reply:
If undefined, it uses the primereact default. But I think it's better to show explicitly what will happen